piptools/scripts/compile.py

```diff
@@ -24,6 +24,7 @@
 
 DEFAULT_REQUIREMENTS_FILE = "requirements.in"
 DEFAULT_REQUIREMENTS_OUTPUT_FILE = "requirements.txt"
+METADATA_FILENAMES = frozenset({"setup.py", "pyproject.toml"})
```

Comment:
Should we also include `setup.cfg` in this list?

Reply:
Great catch! I thought, `setup.py` is required even if you use `setup.cfg` but docs say it is indeed optional:

> If compatibility with legacy builds (i.e. those not using the PEP 517 build API) is desired, a setup.py file containing a setup() function call is still required even if your configuration resides in setup.cfg.

I've updated the line and added a test for setup.cfg as well 👍🏿 

BTW, tox fails if there is no `setup.py`, I'm not the only one who didn't know that 🙃